packages/fxa-admin-server/src/subscriptions/subscriptions.service.ts

```diff
@@ -16,6 +17,20 @@ import {
   StripeFormatter,
 } from './subscriptions.formatters';
 
+/**
+ * List of valid of subscription statuses. This should be all known
+ * stripe subscription types.
+ */
+export const VALID_SUBSCRIPTION_STATUSES: Stripe.Subscription.Status[] = [
+  'active',
+  'canceled',
+  'incomplete',
+  'incomplete_expired',
+  'past_due',
+  'trialing',
+  'unpaid',
+];
```

Comment:
Am a little surprised that no such list already exists in teh repo.

Reply:
I found a reference to `"incomplete_expired"` in packages/fxa-shared/subscriptions/types.ts:133

https://github.com/mozilla/fxa/blob/43e7724fd587db1c7631ecbac5eacce89e71bfc0/packages/fxa-shared/subscriptions/types.ts#L131-L134

Where `Stripe.Subscription.Status` seems to be:

```ts
      type Status =
        | 'active'
        | 'canceled'
        | 'incomplete'
        | 'incomplete_expired'
        | 'past_due'
        | 'trialing'
        | 'unpaid';
```

And the two current references to `Stripe.Subscription.Status` are:

```sh
git grep -n "Stripe.Subscription.Status" | cat

packages/fxa-admin-server/src/subscriptions/subscriptions.formatters.ts:88:  ): Stripe.Subscription.Status | null {
packages/fxa-shared/subscriptions/types.ts:132:      Stripe.Subscription.Status,
```

Reply:
> 



> Looks good but it's fetching more than just "canceled" subscriptions though? e.g. it's also fetching incomplete subscriptions.

That's a fair point. From the ticket I got the feeling that support just wanted to see all the subscriptions. Their semantics didn't quite line up with stripe status semantics though. They were primarily interested in adding 'canceled' and 'inactive' states, which I am sort of assuming would include incomplete.